src/components/AdvertiserName/AdvertiserName.scss

```diff
@@ -3,10 +3,6 @@
     grid-gap: 1.6rem;
     grid-template-columns: min-content auto max-content;
 
-    @include mobile-or-tablet-screen {
-        grid-template-columns: min-content auto;
-    }
-
```

Comment:
Need to remove this since this isnt needed anymore for mobile: 
![Screenshot 2024-10-08 at 3 38 21 PM](https://github.com/user-attachments/assets/95f38046-037c-403a-bb33-1d0151393298)